backend/service/uwfinancepurchases.service.js

```diff
@@ -53,6 +54,13 @@ const updateUWFinancePurchase = async (id, body) => {
         ]
         await Promise.all(emails)
     }
+    if (
+        existingPurchaseTicket.status === 'ORDERED' &&
```

Comment:
should we do a double check? if so we should probably make it consistent across the other transitions just for cleanliness

Reply:
removed for consistency